superduperdb/ext/transformers/training.py

```diff
@@ -182,14 +183,14 @@ class LLMTrainer(TrainingArguments, SuperDuperTrainer):
     max_seq_length: int = 512
     setup_chat_format: bool = False
     log_to_db: bool = True
-    ray_configs: t.Optional[t.Dict] = None
-    on_ray: bool = False
-    ray_address: t.Optional[str] = None
     training_kwargs: t.Dict = dc.field(default_factory=dict)
     num_gpus: t.Optional[int] = None
+    ray_configs: t.Optional[dict] = None
 
     def __post_init__(self, artifacts):
         self.output_dir = self.output_dir or os.path.join("output", self.identifier)
+        if self.num_gpus and 'num_gpus' not in self.compute_kwargs:
+            self.compute_kwargs['num_gpus'] = self.num_gpus
```

Comment:
So this compute_kwargs will be used by ray compute if compute is set to ray, right

Reply:
Yes, the compute_kwargs of trainer is used for training. The compute_kwargs of model is used for prediction. They are different